distribution/test/itests/test-itests-common/src/test/java/ddf/common/test/ServiceManager.java

```diff
@@ -1,665 +1,64 @@
-/**
```

Comment:
Git wasn't too smart about this file change. All I did to the ServiceManager was rename it to ServiceManagerImpl and have it implement my newly created ServiceManager interface to set up a proxy